packages/adapters-library/src/adapters/beefy/tests/snapshots/arbitrum.positions.json

```diff
@@ -0,0 +1,126 @@
+{
+  "blockNumber": 219137472,
+  "snapshot": [
+    {
+      "protocolId": "beefy",
+      "name": "Beefy",
+      "description": "Beefy defi adapter",
+      "siteUrl": "https://beefy.com",
+      "iconUrl": "https://beefy.com/icons/icon-96x96.png",
+      "positionType": "supply",
+      "chainId": 42161,
+      "productId": "moo-token",
+      "assetDetails": {
+        "type": "StandardErc20"
+      },
+      "success": true,
+      "tokens": [
+        {
+          "address": "0xb39a6c02BB32199c5C2e937a9c96Fe8F79E18763",
+          "name": "Moo Aura Gyro aUSDCn/aUSDTn",
+          "symbol": "mooAuraGyroaUSDCn/aUSDTn",
+          "decimals": 18,
+          "balanceRaw": "10250660934841814399n",
+          "type": "protocol",
+          "tokens": [
+            {
+              "address": "0x7CFaDFD5645B50bE87d546f42699d863648251ad",
+              "name": "Static Aave Arbitrum USDCn",
+              "symbol": "stataArbUSDCn",
+              "decimals": 6,
+              "type": "underlying",
+              "balanceRaw": "0n",
+              "balance": 0,
+              "iconUrl": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/arbitrum/assets/0x7CFaDFD5645B50bE87d546f42699d863648251ad/logo.png"
+            },
+            {
+              "address": "0xb165a74407fE1e519d6bCbDeC1Ed3202B35a4140",
+              "name": "Static Aave Arbitrum USDT",
+              "symbol": "stataArbUSDT",
+              "decimals": 6,
+              "type": "underlying",
+              "balanceRaw": "0n",
+              "balance": 0,
+              "iconUrl": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/arbitrum/assets/0xb165a74407fE1e519d6bCbDeC1Ed3202B35a4140/logo.png"
+            }
+          ],
+          "balance": 10.250660934841815
```

Comment:
My colleague spotted the zeros `"balance": 0` are those values coming through correct?

Reply:
Oops, thanks for the review. I will do an additional full manual pass on all test